### PR TITLE
Add diarize emotion

### PR DESCRIPTION
**Pull request by viveknimkarde**

### CODE REVIEW

File: examples/earcningcall_chunk_generator.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+from deepaffects.realtime.util import get_deepaffects_client, chunk_generator_from_file, chunk_generator_from_url
```
Comment (sushant-hiray) [inline examples/earcningcall_chunk_generator.py]: Fix the name of the file

File: examples/diarize_emotion_example.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+# responses is the iterator for all the response values
+for response in responses:
+    print("Received message")
+    print(response)
+
+"""Response.
+    response = {
+        emotion: Emotion identified in the segment,
```
Comment (sushant-hiray) [inline examples/diarize_emotion_example.py]: Where is the information about the user?

File: examples/diarize_emotion_example.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+"""
+Sample implementation which reads audio from a file and splits it into
+segments more than 3 sec
+AudioSegment and yields base64 encoded audio segment objects asynchronously
+"""
+
+"""Stream audio from url or youtube.
+
+responses = client.IdentifyEmotion(
```
Comment (sushant-hiray) [inline examples/diarize_emotion_example.py]: should be `DiarizeEmotion`

**Comment by viveknimkarde** [comment]
@sushant-hiray I have made requested changes can you please review and push package to pypi

**Pull request merged**